tests/b017.py

```diff
@@ -1,10 +1,12 @@
 """
 Should emit:
-B017 - on lines 20
+B017 - on lines 24 and 26.
 """
 import asyncio
 import unittest
 
+import pytest
+
```

Comment:
Good question. Not sure how this is being found. Agreed on adding to dev optional dependencies, though it would be good to know where it's currently being taken from.